CHANGELOG.md

```diff
@@ -12,6 +12,19 @@
 
 ---
 
+[//]: # (START/v1.4.1)
+# v1.4.1
+
+Credits to @mjpieters for the contribution! :rocket:
+
+## Fixes
+ * Undeclared dependency: six (in generated code). {#85}
+ * Leave JSON encoding to HTTPX. {#87}
+ * Correct function name in documentation. {#90}
+ * Update example main.py to handle 'n' input. {#91}
```

Comment:
`[question]`

These two changelogs are more related to keeping the documentation in good shape. These don't directly impact the users in terms of how they use the SDK, do they? 🤔 

However, I do see that in previous releases we mentioned any improvements to the documentation, which makes me think that this is fine to include. 

Let me know what you think. 😄

Reply:
In general, I'd say, that if the change impacts the users or the change is made by an external contributor it is worth to mention. If this is something small and is done by the 1Password team (fixing a typo or rephrasing the sentence), let's not include it.

Let's keep these 2 in the current PR.